sdk/python/setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = oomstore
+version = 0.0.1
+author = oom
+author_email = author@example.com
```

Comment:
I'm not sure of the value of author and author_email

Reply:
let's do evan.chanyiksan@gmail.com for now.

Once we setup @oom.ai emails, we can change